### DISCUSSION TITLE
How to import credentials from the old WebSSH app?

### DISCUSSION THREAD

**Discussion by reinhart1010** [isontheline/pro.webssh.net#856] [Q&A] [answered]
Hi, I have previously installed a very old version of WebSSH (as of early 2020, imported to recent device by iCloud backup). How do I import my credentials from the old app to the new one?

![image](https://user-images.githubusercontent.com/17312341/228753023-b347506a-6f33-4392-9ac2-2b38ab0039e8.png)

**Comment by isontheline** [comment, marked as answer]
Hi @reinhart1010 👋

This legacy lacks of export feature if I remember correctly.

The only way I think should be to explore your iPhone backup on your computer (with a tool to extract) in order to copy the "webssh.db" database located under "Application Support" of the WebSSH app.
